src/components/ScrollingText/ScrollingText.css

```diff
@@ -2,9 +2,45 @@
   grid-column: 1 / 2;
   grid-row: 1 / 3;
   color: gold;
-  font-size: 1.4rem;
+  font-size: 2.5rem;
   text-align: justify;
-  padding: 20px;
+  overflow: hidden;
+  perspective: 400px;
+  perspective-origin: 50% 30%;
 }
 
+.title {
+  margin-bottom: 0;
+  text-align: left;
```

Comment:
Add a `line-height` to fix the larger space between title that run onto two lines. Looks a little funky right now.

Reply:
Done!